data/sdss/Makefile

```diff
@@ -10,46 +10,48 @@ FIELD_STRIPPED := $(shell echo $(FIELD) | sed 's/^0*//')
 RUN6 := $(shell printf '%06d' $(RUN_STRIPPED))
 FIELD4 := $(shell printf '%04d' $(FIELD_STRIPPED))
 
+DOWNLOAD_DIR := $(shell pwd)
 SUBDIR2 := $(RUN_STRIPPED)/$(CAMCOL)
 SUBDIR3 := $(SUBDIR2)/$(FIELD_STRIPPED)
 
-all : $(SUBDIR3)/fpM-$(RUN6)-u$(CAMCOL)-$(FIELD4).fits \
-      $(SUBDIR3)/fpM-$(RUN6)-g$(CAMCOL)-$(FIELD4).fits \
-      $(SUBDIR3)/fpM-$(RUN6)-r$(CAMCOL)-$(FIELD4).fits \
-      $(SUBDIR3)/fpM-$(RUN6)-i$(CAMCOL)-$(FIELD4).fits \
-      $(SUBDIR3)/fpM-$(RUN6)-z$(CAMCOL)-$(FIELD4).fits \
-      $(SUBDIR3)/frame-u-$(RUN6)-$(CAMCOL)-$(FIELD4).fits \
-      $(SUBDIR3)/frame-g-$(RUN6)-$(CAMCOL)-$(FIELD4).fits \
-      $(SUBDIR3)/frame-r-$(RUN6)-$(CAMCOL)-$(FIELD4).fits \
-      $(SUBDIR3)/frame-i-$(RUN6)-$(CAMCOL)-$(FIELD4).fits \
-      $(SUBDIR3)/frame-z-$(RUN6)-$(CAMCOL)-$(FIELD4).fits \
-      $(SUBDIR2)/photoField-$(RUN6)-$(CAMCOL).fits \
+all : $(DOWNLOAD_DIR)/$(SUBDIR3)/fpM-$(RUN6)-u$(CAMCOL)-$(FIELD4).fits \
```

Comment:
Perhaps we should just delete this file if `sdss_download.py`  is working well. It's easier not to maintain two ways of doing the same thing.